src/google-classroom-extractor/README.md

```diff
@@ -46,15 +47,18 @@ to `.env`, and adjust to your desired parameters. Supported parameters:
 
 | Description | Required | Command Line Argument | Environment Variable |
 | ----------- | -------- | --------------------- | -------------------- |
-| The email address of the Google Classroom admin account. | yes | -a or --classroom-account | CLASSROOM_ACCOUNT |
-| The log level for the tool. | no (default: INFO) | -l or --log-level | LOG_LEVEL |
-| The output directory for the generated csv files. | no (default: data/) | -s or --usage-start-date | OUTPUT_PATH |
-| Start date for usage data pull in yyyy-mm-dd format. | no (default: today) | -s or --usage-start-date | START_DATE |
-| End date for usage data pull in yyyy-mm-dd format. | no (default: today) | -e or --usage-end-date | END_DATE |
+| The email address of the Google Classroom admin account. | yes | `-a` or `--classroom-account` | CLASSROOM_ACCOUNT |
+| The log level for the tool. | no (default: INFO) | `-l` or `--log-level` | LOG_LEVEL |
+| The output directory for the generated csv files. | no (default: data/) | `-s` or `--usage-start-date` | OUTPUT_PATH |
+| Start date*, yyyy-mm-dd format | no (default: today) | `-s` or `--usage-start-date` | START_DATE |
+| End date*, yyyy-mm-dd format | no (default: today) | `-e` or `--usage-end-date` | END_DATE |
```

Comment:
I think adding the date format for the parameters, is a good idea

Reply:
Format is already there...

Reply:
Sorry I did not see it... 😅